Tests/scripts/validate_premium_packs.py

```diff
@@ -0,0 +1,266 @@
+                             success_message=f"{index_file_path} file was found valid")
+    return packs_are_valid, index_data
+
+
+def get_paid_packs(client: demisto_client, request_timeout: int = 999999):
+    """Get premium packs from client.
+
+    Trigger an API request to demisto server.
+    Request is identical to checking the premium box through the marketplace GUI.
+
+    Args:
+        client: The demisto client to the preform request on.
+        request_timeout: Timeout of API request
+
+    Returns:
+        Dict of premium packs as found in the server.
+        Return None if no premium packs were found.
+    """
+    request_data = \
+        {
+            'page': 0,
+            'size': 50,
+            'sort':
+                [{
+                    'field': 'updated',
+                    'asc': False
+                }],
+            'general': ["generalFieldPaid"]
+        }
+
+    logging.info(f'Getting premium packs from server {client.api_client.configuration.host}:\n')
+
+    # make the pack installation request
+    response_data, status_code, _ = demisto_client.generic_request_func(client,
+                                                                        path='/contentpacks/marketplace/search',
+                                                                        method='POST',
+                                                                        body=request_data,
+                                                                        accept='application/json',
+                                                                        _request_timeout=request_timeout)
+
+    if status_code == 200:
+        logging.debug(f'Got response data {response_data}')
+        response = ast.literal_eval(response_data)
+        logging.debug(f'Response dict is {response}')
+        logging.info('Got premium packs from server.')
+        return response["packs"]
+
+    result_object = ast.literal_eval(response_data)
+    message = result_object.get('message', '')
+    logging.error(f'Failed to retrieve premium packs - with status code {status_code}\n{message}\n')
+    return None
+
+
+def verify_server_paid_packs_by_index(server_paid_packs, index_data_packs):
+    """Compare two pack dictionaries and assert id's and prices are identical.
+
+    Log errors if the lists differ.
+
+    Args:
+        server_paid_packs: Dictionary of packs to check.
+        index_data_packs: Dictionary of packs to compare to.
+
+    Return:
+        True if all packs are identical, False otherwise.
+    """
+    # Sorting both lists by id
+    sorted_server_packs = sorted(server_paid_packs, key=lambda pack: pack['id'])
+    sorted_index_packs = sorted(index_data_packs, key=lambda pack: pack['id'])
+
+    packs_identical = True
+    # Checking lists are the same.
+    for (server_pack, index_pack) in zip(sorted_server_packs, sorted_index_packs):
```

Comment:
Add an info log like `Validating pack {pack_id}`

Reply:
I think it will make the logs a little long. Check out the Validate premium packs stage at 
https://app.circleci.com/pipelines/github/demisto/content/47595/workflows/288f8a53-308f-42e7-9605-72f1bf721acc/jobs/203927
Let me know if you still think its necessary